api/mapping/test_views.py

```diff
@@ -64,17 +64,19 @@ def test_dataset_returns(self):
         )
         # Get the response data
         response_data = self.view(request).data
+        expected_objs = [
+            self.public_dataset1.id,
+            self.public_dataset2.id,
+            self.restricted_dataset.id,
+        ]
 
         # Assert user1 can only public_dataset1, public_dataset2
         # and restricted_dataset
+        self.assertEqual(len(response_data), len(expected_objs))
```

Comment:
Rather than comparing length and asserting contents, could we rather more directly do 
```
assertEqual(sorted(expected_objs), sorted(response_data))
```
?